rrdp_iostat.ml

```diff
@@ -422,6 +450,10 @@ module Stats_value = struct
 		let (++) = Int64.add in
 		List.fold_left (fun acc v ->
 			{
+				rd_bytes = acc.rd_bytes ++ v.rd_bytes;
+				rd_avg_usecs = acc.rd_avg_usecs ++ v.rd_avg_usecs;
```

Comment:
This sum is for the sumup function in stats_value module - I think this is probably to get the total of all stats.